src/admin.rs

```diff
@@ -361,11 +361,11 @@ fn change_log_level(reset: bool, level: &str) -> Response<Full<Bytes>> {
     match tracing::level_filters::LevelFilter::from_str(level) {
         Ok(level_filter) => {
             // Valid level, continue processing
-            tracing::info!("Parsed level: {:?}", level_filter);
+            tracing::debug!("Parsed level: {:?}", level_filter);
```

Comment:
nit: this API isn't going to be called often (outside of debugging), could probably leave at `info` level

Reply:
cc/ @keithmattix since he suggested the change initially. Any strong opinions against leaving as info?

Reply:
No strong opinions; we can always lower the level later. Feel free to change it back to info